.github/workflows/pulumi-aws-tests.yml

```diff
@@ -33,14 +33,14 @@ jobs:
     - uses: syphar/restore-virtualenv@v1
       id: pulumi-aws-tests-cache-virtualenv
       with:
-        requirement_files: pulumi/aws/requirements.txt
+        requirement_files: config/requirements.txt
 
     - uses: syphar/restore-pip-download-cache@v1
       if: steps.pulumi-aws-tests-cache-virtualenv.outputs.cache-hit != 'true'
 
     - run: ./setup_venv.sh
-      working-directory: pulumi/aws
+      working-directory: bin
```

Comment:
Is this the right directory? It seems substantial to change from the root directory to the script directory.

Reply:
Wasn't 100% sure where I was going there and what made sense; adjusted, works.